documents/documents.html

```diff
@@ -130,10 +131,17 @@ <h2>Installation Guide</h2>
           </p>
           <div class="mt-2 code-snippet">
             <iframe
-  src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fdreamui-demo.netlify.app%252Fdreamui-v1.css%2522%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
-  style="width: 839px; height: 76px; border:0; transform: scale(1); overflow:hidden;"
-  sandbox="allow-scripts allow-same-origin">
-</iframe>
+              src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fdreamui-demo.netlify.app%252Fdreamui-v1.css%2522%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
+              style="
```

Comment:
the scroll bar inside the code snippet container is not very attractive, you can remove it by removing this inline style attribute altogether and give this width of 100% of parent container so that the scroll bar will be at the end. refer to this https://github.com/bhargaviRdy/vibe-UI/blob/dev/docs.css#L66 if you need help